WPFHospitalEditor/Controller/ExaminationServerController.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using WPFHospitalEditor.Controller.Interface;
+using WPFHospitalEditor.Service;
+using WPFHospitalEditor.Service.Interface;
+
+namespace WPFHospitalEditor.Controller
+{
+    public class ExaminationServerController : IExaminationServerController
+    {
+        IExaminationServerService examinationServerService = new ExaminationServerService();
+        public string ScheduleExamination(DateTime startTime, int doctorId, int patientId)
```

Comment:
What should the returned value represent? It is probably better to return the scheduled examination or null if the scheduling failed.

Reply:
fixed